### DISCUSSION TITLE
See DialogFlow Tutorials

### DISCUSSION THREAD

**Discussion by RobertoRuizUnda** [fonoster/yaps#446] [General]
In the documentation there is a segment that says Tutorials:
- Using Fonoster with Google Voice APIs
- Connecting Dialogflow and Fonoster
- Use of Javascript SDKs

I need to test a fixed IP number, register it with Fonoster and later integrate with DialogFlow

The first tutorial (Using Fonoster with Google Voice APIs) only applies when it is the Fonoster Cloud?
Does the second tutorial (Connecting Dialogflow and Fonoster) only apply with Fonoster running locally?

Or to carry out a test, the three tutorials must be carried out?

**Comment by psanders** [comment]
Hey @RobertoRuizUnda 

It sounds looks your ultimate goal is to connect calls from the PSTN to a DialogFlow bot. Is that right? I recommend breaking the problem into the following tasks.

1. Create a Dialogflow ES bot
2. Connect to Fonoster and test from the browser (You can test the bot without a real number)
3. Configure a Provider and Number and point to your Dialogflow bot

You can steps 1 and 2 by following @YuriCodes [video tutorial](https://www.youtube.com/watch?v=m6B_9lU4iSw). We can then focus on the last step. Please, if you get stuck let me know but try following this steps in sequence.

**Reply by RobertoRuizUnda** [reply]
Thank you very much for the reply.
If you successfully complete steps 1 and step 2.
A question about step number 3 (point to your Dialogflow bot) there are two ways to do it:
a) Configure Provider and Number from Fonoster Cloud. (local server voice and cloud CLI)
b) Configure Provider and Number from Fonoster all locally.
I am right?

**Reply by psanders** [reply]
I'm locking this conversation to continue on Discord.
